app/src/main/java/org/fossasia/openevent/dbutils/DatabaseOperations.java

```diff
@@ -695,4 +696,21 @@ public void insertQuery(String query, DbHelper mDbHelper) {
         db.endTransaction();
     }
 
+    public void addToDb(int id) {
+        String query_normal = "INSERT INTO %s VALUES ('%d');";
```

Comment:
you should normalize the `%s` because some one may inject a string to it and cause SQL injection. http://stackoverflow.com/questions/7336563/android-sqlite-insert